src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -698,6 +704,10 @@ bool fx_muxer_t::resolve_sdk_dotnet_path(const pal::string_t& own_dir, const pal
     }
     
     trace::verbose(_X("It was not possible to find any SDK version"));
```

Comment:
This does not necessarily apply when we're looking for a specific version, so we should log either this or new message, but not both.